### DISCUSSION TITLE
Duplicate transactions being created through importer

### DISCUSSION THREAD

**Discussion by firsttiger** [firefly-iii/firefly-iii#4817] [Firefly III]
**Bug description**
I am running Firefly III CSV Importer version 2.5.2, and my problem is:

I am importing a csv file from my bank and the csv seems to be importing duplicates even though I have duplicate detection set to content-based

**Steps to reproduce**
Using the CSV Importer, I import the file and select a pre-made configuration file
Import file
Go to account page
Duplicates appear on the account

**Extra info**

**Bonus points**

- I searched and nobody reported this bug before
- I have added a stack trace from my log files 
- I have added a screenshot.
- I was able to replicate it on the demo site https://demo.firefly-iii.org/

**Comment by JC5** [comment]
Could you share some example lines and your configuration?
